test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -1154,4 +1155,50 @@ public void addHighlighting() throws Exception {
             assertThat("Incorrect highlighting", res, is(result2));
         }
     }
+    @Test
+    public void sortingResults() throws Exception {
+        //type is SimpleNumber
+        GUID guid1 = new GUID("WS:11/1/2");
+        GUID guid2 = new GUID("WS:12/2/2");
+        GUID guid3 = new GUID("WS:13/3/2");
+        prepareTestLookupInKey(guid1, guid2, guid3);
+        //type is Simple
+        GUID guid4 = new GUID("WS:14/1/2");
+        GUID guid5 = new GUID("WS:15/2/2");
+        GUID guid6 = new GUID("WS:16/3/2");
+        prepareTestMultiwordSearch(guid5, guid6, guid4);
+        List<String> empty = new ArrayList<>();
```

Comment:
Should interleave the type and number so that the type sort is exercised. right now the type sort is the same as the number sort.